imports/plugins/core/orders/client/containers/invoiceContainer.js

```diff
@@ -223,9 +223,9 @@ class InvoiceContainer extends Component {
     const paymentMethodId = orderBillingInfo.paymentMethod && orderBillingInfo.paymentMethod.paymentPackageId;
     const paymentMethodName = orderBillingInfo.paymentMethod && orderBillingInfo.paymentMethod.paymentSettingsKey;
     const paymentMethod = Packages.findOne({ _id: paymentMethodId });
+    const refund = _.get(paymentMethod.settings[paymentMethodName].support, "refund");
```

Comment:
There's another problem in Line 273:
![invoiceactions_js_-_reaction_-____github_reactioncommerce_reaction__and_reactioncommerce_reaction_-_gitter](https://user-images.githubusercontent.com/1733229/34771016-59182b08-f603-11e7-9e0a-cfb733992b12.jpg)

Reply:
@iakhator  Also in shippingInvoice.js, line 130 ff

Reply:
@iakhator See also exampleSettingsForm.js from payments-example:
![examplesettingsform_js_-_reaction_-____github_reactioncommerce_reaction_](https://user-images.githubusercontent.com/1733229/34771267-3daed01e-f604-11e7-909c-56cf673a61c0.jpg)